app/api/v1/guardian/guardian.py

```diff
@@ -41,13 +41,22 @@ def create_guardian(request: GuardianRequest) -> Guardian:
     Create a guardian for the election process with the associated keys
     """
     election_keys = generate_election_key_pair(
+        request.id,
+        request.sequence_order,
         request.quorum,
         int_to_q_unchecked(request.nonce) if request.nonce is not None else None,
     )
     if request.auxiliary_key_pair is None:
-        auxiliary_keys = generate_rsa_auxiliary_key_pair()
+        auxiliary_keys = generate_rsa_auxiliary_key_pair(
+            request.id, request.sequence_order
+        )
     else:
-        auxiliary_keys = request.auxiliary_key_pair
+        auxiliary_keys = EG_AUX.AuxiliaryKeyPair(
```

Comment:
**#NOTE1: Handling Request JSON objects vs original ElectionGuard Models**
I think this should get moved to a mapping file between the models. This is probably going to come up a bunch with serialization. 

I'm thinking in the model file itself it has the mapping to the electionguard equivalent.

Reply:
`auxiliary_key_pair.convert_to_eg()` or something like that.

This should also be using the existing function to do this:
`read_json_object(request.auxiliary_key_pair , EG_AUX.AuxiliaryKeyPair)`

Reply:
Can also be done with EG_AUX.AuxiliaryKeyPair.from_json_object(request.auxiliary_key_pair)